napari/view_layers.py

```diff
@@ -25,6 +25,7 @@ def view_image(
     title='napari',
     ndisplay=2,
     order=None,
+    axis_labels=None,
```

Comment:
you'll ultimately want to add `axis_labels` as a keyword arg to all our `view_*` methods